src/index.js

```diff
@@ -1,53 +1,70 @@
 'use strict'
 
-var invert = require('invert-hash')
+const bs58 = require('bs58')
 
-var mh = module.exports = function () {
-  if (arguments.length === 1) {
-    return mh.decode.apply(this, arguments)
-  } else if (arguments.length > 1) {
-    return mh.encode.apply(this, arguments)
+const cs = require('./constants')
+
+exports.toHexString = function toHexString (m) {
+  if (!Buffer.isBuffer(m)) {
+    throw new Error('must be passed a buffer')
+  }
+
+  return m.toString('hex')
+}
+
+exports.fromHexString = function fromHexString (s) {
+  return new Buffer(s, 'hex')
+}
+
+exports.toB58String = function toB58String (m) {
```

Comment:
we've been using the `toSomething` as obj.toNewFormat. Since this is a utility function and not something available on the object, I would prefer and argue it would be less confusing if instead of 'to' we use the word 'convert' or 'transform'